python/kserve/kserve/protocol/dataplane.py

```diff
@@ -351,7 +350,7 @@ async def generate(
 
         Args:
             model_name (str): Model name.
-            request (bytes|GenerateRequest): Generate Request body data.
+            request (bytes|GenerateRequest): Generate Request / ChatCompletion Request body data.
```

Comment:
Is this generate function still used, I think it uses the openai data plane now right?